JobRecruiting/JobRecruiting/src/main/webapp/WEB-INF/tags/jobCard.tag

```diff
@@ -34,6 +34,18 @@
         <p class="card-text mt-4">${description}</p>
         <div class="card-text">
             <p class="card-text mt-1 ">Salary: <span style="font-weight:bold">${salary}$</span></p>
+            <c:choose>
+                   user.interviews.contains(id)
+                <c:when test="${pageContext.request.getRemoteUser()==null}">
+                    <a href="${pageContext.request.contextPath}/ApplyForJob?jobid=${id}" class="mt-1 me-3 btn  btn-success fs-5">
+                        Apply
+                    </a>
+                </c:when>
+                <c:otherwise>
+                    <p>Applied</p>
+                </c:otherwise>
+            </c:choose>
```

Comment:
o sa schimb asta cand o sa am chestiile legate de user intrview facute

Reply:
Sunt facute o parte din ele. Poti sa le iei din branchul meu de interviews. Acolo e tot proiectul updatat, in afara de ce a ramas prin PRs, deci nu e nici partea asta a ta.